scripts/eurostat/gdp/import_data.py

```diff
@@ -81,9 +86,23 @@ def clean_data(self):
                              "Please check you are calling preprocess_data "
                              "before clean_data.")
         self.clean_df = self.preprocessed_df[self.DESIRED_COLUMNS]
+        self.clean_df = self.clean_df[~self.clean_df['geo'].isin(['EU27_2020',
```

Comment:
Please add a comment explaining the human meaning why these are being removed.

Reply:
Done.